doc/man3/SSL_CTX_set_options.pod

```diff
@@ -211,6 +211,12 @@ not propose, and servers will not accept the extension.
 Disable all renegotiation in TLSv1.2 and earlier. Do not send HelloRequest
 messages, and ignore renegotiation requests via ClientHello.
 
+=item SSL_OP_LEGACY_SSL_EOF
+
+Some TLS implementations do not send close_notify on shutdown. Such behaviour
+is treated as error since OpenSSL 3.0. Enabling this option allows to treat
+such behaviour as normal. See more details in L<SSL_CONF_cmd(3)>.
```

Comment:
I'd copy the wording from -legacy_ssl_eof here.

Reply:
Fixed. Thanks!